doc/source/whatsnew/v1.0.0.rst

```diff
@@ -39,8 +39,59 @@ Backwards incompatible API changes
 
 .. _whatsnew_1000.api.other:
 
-- :class:`pandas.core.groupby.GroupBy.transform` now raises on invalid operation names (:issue:`27489`).
--
+Groupby.transform(name) accepts only aggregation names
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:meth:`DataFrameGroupBy.transform` and :meth:`SeriesGroupBy.transform` are used to either invoked
```

Comment:
This seems a bit wordy. Can we simplify?  For example, I think this entire first sentence can be removed.

The key point to convey is that GroupBy.transform now ensures that the name provided is an aggregation, correct? Anything else?

Reply:
My feeling is that there is some confusion as to what `transform` is `supposed` to do. For example, in the issue I opened to discuss agg vs. transform, I understood you to say you you found the broadcasting behavior of `transform('rank')` useful. 

So, I wanted to give a direct explicit description of what the role of `transform` is and that was the purpose of the wordy first sentence, which I've removed now.